.github/workflows/swift-toolchain.yml

```diff
@@ -27,6 +25,24 @@ on:
         description: 'Code Sign'
         default: false
         type: boolean
+
+      default_runner:
```

Comment:
Can we move these changes to upstream?

Reply:
Done: https://github.com/compnerd/swift-build/pull/747